apps/studio/prisma/schema.prisma

```diff
@@ -125,6 +125,9 @@ model Site {
   /// @kyselyType(PrismaJson.SiteJsonConfig)
   /// [SiteJsonConfig]
   config      Json
+  /// @kyselyType(PrismaJson.SiteThemeJson)
+  /// [SiteThemeJson]
+  theme       Json?
```

Comment:
nit: wondering if it'd be better to remove this since we can always just give a default theme but not very importatnt since they are functionally equivalent